test/integration/kubernetes_deploy_test.rb

```diff
@@ -1006,4 +1007,19 @@ def test_raise_on_yaml_missing_kind
       "  datapoint: value1"
     ], in_order: true)
   end
+
+  def test_crd_can_be_successful
+    assert_deploy_success(deploy_fixtures("hello-cloud", subset: ["crd.yml"]))
```

Comment:
I don't think this test adds anything over the `full_hello` test you modified above.

Reply:
It doesn't, but this file is actually from the base branch (https://github.com/Shopify/kubernetes-deploy/pull/306/files#diff-43c27b92ee93912981b34f7d8b622714R1011) where it now is a legit test.  I'll fix all of the up once that PR merges.